readthedocs/config/tests/test_config.py

```diff
@@ -222,15 +222,15 @@ def test_python_pip_install_default():
     assert build.pip_install is False
 
 
-def describe_validate_python_extra_requirements():
+class ValidatePythonExtraRequirements:
 
-    def it_defaults_to_list():
+    def it_defaults_to_list(self):
```

Comment:
Also in the methods, we need to start the name with `test_`

Reply:
Right! My bad.

Reply:
should I use test_it_defaults_to_list or test_defaults_to_list?